setupdocs.md

```diff
@@ -0,0 +1,182 @@
+
+# Car-Bon System: Technical Setup Guide
+
+## Prerequisites
+
+Before beginning the installation, ensure you have:
+
+-   Administrator access to the project repository of github
+-   Windows operating system
+-   Basic understanding of command-line interfaces
+
+## Version Control Setup
+
+1.  Install Git
+    -   Download and install from the official website
+    -   Configure with your credentials
+2.  Clone Project Repository
+    
+    bash
+    
+    Copy
+    
+    `git clone https://github.com/gunnnu/Car-Bon_UI.git`
+    
+    **Note:** Repository access requires admin invitation
+
+## Windows Subsystem for Linux (WSL) Configuration
+
+1.  Open PowerShell as Administrator
+    
+    powershell
+    
+    Copy
+    
+    `wsl --install`
+    
+2.  Complete Ubuntu installation and create user account
+
+## Python Environment Preparation
+
+1.  Install Python 3.9+
+    
+    bash
+    
+    Copy
+    
+    `python3 --version`
+    
+2.  Configure Python System
+    
+    bash
+    
+    Copy
+    
+    `sudo  apt  install python-is-python3 sudo update-alternatives --install /usr/bin/python python $(readlink -f $(which python3))  3`
+    
+3.  Package Management
+    
+    bash
+    
+    Copy
+    
+    `sudo  apt update sudo  apt  install python3-pip pip install poetry`

```

Comment:
This should just mention the install of poetry and not pip also this could be a step in configuration and not a separate section on its own